cmd/p2p/p2p.go

```diff
@@ -45,6 +45,7 @@ func (app *P2PApp) Cleanup() {
 func (app *P2PApp) Start(cmd *cobra.Command, args []string) {
 	// init p2p services
 	log.JSONLog(true)
+        log.DebugMode(true)
```

Comment:
please run gofmt on file or just fix indentation as this will not pass travis' checks